include/jsonbuilder/JsonBuilder.h

```diff
@@ -145,15 +205,14 @@ class PodVectorBase
     static size_type GetNewCapacity(size_type minCapacity, size_type maxCapacity);
 
     /*
-    Calls HeapAlloc or HeapReAlloc. If allocation fails, throw bad_alloc.
+    Calls malloc. If allocation fails, throw bad_alloc.
```

Comment:
Note: The Realloc implementation was very broken if zeroInitializeMemory is used. Fix by just implementing malloc, not realloc.